lib/components.js

```diff
@@ -363,7 +363,9 @@ class DatePicker extends Component {
       'minimumDate',
       'minuteInterval',
       'mode',
-      'timeZoneOffsetInMinutes'
+      'timeZoneOffsetInMinutes',
+      'background',
```

Comment:
I thought it should be in the same level as `minuteInterval` or `minimumDate`, relative to the RN component. The format function or the mode are relative to the `datepicker` function, so I thought the background should be out there.